src/modules/profiles/service/ping.serivce.ts

```diff
@@ -0,0 +1,46 @@
+import { PrismaClient } from '@prisma/client';
+import { sendSmsService } from 'src/utils/emailSmsService';
+import { NextApiRequestWithTenant } from '../../../lib/next-connect/index';
+
+export default class PingService {
+  static send = async (
+    tenantPrisma: PrismaClient,
+    userId: string,
+    req: NextApiRequestWithTenant,
+  ) => {
+    const newPing = await tenantPrisma.ping.create({
+      data: {
+        message: req.body.message,
+        pingerInfo: {
+          connect: {
+            userId: req.user.id,
+          },
+        },
+        pingAlumniInfoId: userId,
+      },
+    });
+
+    const userInformation = await tenantPrisma.information.findUnique({
+      where: { userId },
+      include: {
+        alumClass: {
+          include: {
+            grade: true,
+          },
+        },
+      },
+    });
+
+    console.log(userInformation);
+
+    if (userInformation?.phone) {
+      await sendSmsService({
+        to: `+84${userInformation?.phone}`,
```

Comment:
I have no idea how it works. But if my phone number is `03344xxxx`, so the destination of the message will be `+8403344xxxx`, right? Will the message be sent to the right destination?

Reply:
yeah i was concerned about that too. But ye, it works fine! No worries